src/remediations/fifi.js

```diff
@@ -63,13 +63,9 @@ exports.checkSmartManagement = async function (remediation, smart_management) {
 };
 
 exports.checkRhcEnabled = async function () {
-    const rhcStates = await configManager.getCurrentState();
+    const rhcProfiles = await configManager.getCurrentProfile();
 
-    if (rhcStates.state.remediations === DISABLED) {
-        return false;
-    }
-
-    return true;
+    return !! rhcProfiles.remediations
```

Comment:
By looking at the `getCurrentProfile` method, it might return a `null`. Probably use a safe navigation here.

Don't forget to end statements with a semicolon `;`. :slightly_smiling_face: